api-reference/listing/task-webhook.mdx

```diff
@@ -0,0 +1,221 @@
+---
+title: "Create Task Webhook Event"
+api: "POST https://www.example.com/webhook/task_creation"
+description: "This endpoint for receiving task creation webhook event."
+---
+
+This endpoint allows you to receive a task creation event from Adam.
+
+When you receive a task creation event, you must send a 2xx response.
+If you do not send a 2xx response Adam will resend a task creation event until 3 times failure.
+
+### Header
+
+<ParamField header="Authorization" type="string" default="none" required>
+  The authentication token for your request
+</ParamField>
+
+### Body
```

Comment:
I'd make it clearer here that this is what the task webhook will send to the URL that the client provides. I would also add a section describing how they could subscribe to this webhook, I've added a section to the Notion page.